packages/gatsby-theme/src/components/Grid/Grid.js

```diff
@@ -48,14 +48,8 @@ const Grid = ({
 }
 
 Grid.propTypes = {
-  columns: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
-    PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  ]),
-  proportion: PropTypes.oneOfType([
-    PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string])),
-    PropTypes.oneOfType([PropTypes.string]),
-  ]),
+  columns: PropTypes.node,
+  proportion: PropTypes.node,
```

Comment:
We should be careful with `PropTypes.node`. In this case, having a type node here would mean we could pass something like `columns={<div>hello</div}`.